src/datafactory/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = "1.0.3"
+VERSION = "1.0.4"
 try:
```

Comment:
The most recent version for datafactory is `1.0.2` and `1.0.3` failed in releasing because the version in `src/datafactory/azext_datafactory/manual/version.py` didn't get changed in last pr, as you may notice.

Based on this, two suggestions we recommend now for our extension developers.
1. use `VERSION` in `src/datafactory/setup.py` to manage the versioning uniquely
2. remove the code line https://github.com/Azure/azure-cli-extensions/blob/main/src/datafactory/setup.py#L14-L17 to avoid any future potential missing in versioning management.